ostia-operator/Brewfile

```diff
@@ -0,0 +1,2 @@
+brew 'operator-sdk'
```

Comment:
Can you pin a version?

Reply:
I don't think so. But you can't do that in brew anyway. There is always just the latest.

Reply:
Ok.. let's see how it goes :)